README.md

```diff
@@ -20,6 +20,7 @@ Multiple people and organizations are joining efforts to create a single Externa
 - [Yandex Lockbox](https://external-secrets.io/provider-yandex-lockbox/)
 - [Gitlab Project Variables](https://external-secrets.io/provider-gitlab-project-variables/)
 - [Alibaba Cloud KMS](https://www.alibabacloud.com/product/kms) (Docs still missing, PRs welcomed!)
+- [Oracle Vault](https://www.oracle.com/uk/security/cloud-security/key-management/) 
```

Comment:
You can put the actual doc that you created here, the final link will be: https://external-secrets.io/provider-oracle-vault